src/components/Rating.js

```diff
@@ -2,107 +2,107 @@ import StarFull from '../images/star_full.svg';
 import StarEmpty from '../images/star-empty.svg';
 import StarHalf from '../images/star-half.svg';
 
-export default function Rating({ stars }) {
+export default function Rating({ stars, height }) {
```

Comment:
See if you can figure out how build the stars programmatically instead of using brute force.